Makefile

```diff
@@ -463,6 +463,12 @@ release-only:
 		echo 'Please update Added: tags in the documentation first.' ; \
 		exit 1 ; \
 	fi
+	@if $(grep -IPqrs '^>>>>>>> [0-9A-Fa-f]+' benchmark deps doc lib src test tools) || \
```

Comment:
I'd also check for `^<<<<<<<` (but not `^=======` because that's going to match a gazillion section markers in documentation.)